auth-web/tests/unit/components/AccountAuthorizationRequest.spec.ts

```diff
@@ -0,0 +1,127 @@
+import '../test-utils/composition-api-setup' // important to import this first
+import Vue, { VueConstructor } from 'vue'
+import { Wrapper, createLocalVue, mount } from '@vue/test-utils'
+import {
+  orgsDetailsByAffiliationEmptyResponse,
+  orgsDetailsByAffiliationMultipleItemsResponse,
+  orgsDetailsByAffiliationSingleItemResponse
+} from '../test-utils'
+
+import AccountAuthorizationRequest from '@/components/auth/manage-business/AccountAuthorizationRequest.vue'
+import Vuetify from 'vuetify'
+import Vuex from 'vuex'
+import { axios } from '@/util/http-util'
+import flushPromises from 'flush-promises'
+import sinon from 'sinon'
+
+Vue.use(Vuetify)
+Vue.use(Vuex)
```

Comment:
I'd recommend setting it up like this:
https://github.com/bcgov/business-edit-ui/blob/main/tests/unit/setup.ts
https://github.com/bcgov/business-edit-ui/blob/0007c737e64ce1ae1aee2db1341da922e039f089/jest.config.js#L11

So we don't have a bunch of imports and Vue use at the top of each unit test

Reply:
I tried to see if I can quickly update current setup, but run in some issues. 
So like you suggested, lets worry later

Also, maybe we can connect and see what approach I should do for the next UI parts I will need to add. I could possibly update this test file retroactively too.